core/src/avm2/globals/flash/net.rs

```diff
@@ -1,5 +1,7 @@
 //! `flash.net` namespace
 
+pub mod file_reference;
```

Comment:
can't these two be defined in AS too?

Reply:
Temporary yes, but in the future when someone will implement the browse or download method they will need to do OS or browser operations afaik.

Reply:
Sure, but we still keep the class defined in ActionScript - when we actually implement it, we can annotate the methods with `native function`, so that they can still be implemented in Rust.
Baiscally, the .as class replaces `pub fn create_class(...)` boilerplate; ideally long-term we'd like to convert all our class declarations from Rust to AS.

Reply:
Ok now it's more clear because I did not see many in AS and I thought it was for perf reason

Reply:
No, the real reason is that we added AS classes way later, and some of these existing Rust-defined classes are slightly nontrivial to convert, so nobody did them yet ;)

Reply:
Ok! Fixed as requested